lib/logflare/cluster/utils.ex

```diff
@@ -24,4 +25,9 @@ defmodule Logflare.Cluster.Utils do
   def rpc_multicall(mod, func, args, timeout \\ 5_000) do
     :rpc.multicall(node_list_all(), mod, func, args, timeout)
   end
+
+  @spec erpc_call(node(), function(), non_neg_integer()) :: term()
+  def erpc_call(node, func, timeout \\ 5_000) do
```

Comment:
We can just call it rpc_call, since we don't need to distinguish between them from caller pov

Reply:
Yes, done in https://github.com/Logflare/logflare/pull/2785 :pray: